seeds/AdvancedSettings.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+[{"id":1,"title":"Community recommendations","description":"Show posts that are recommended by other community members.","group":"advanced","defaultEnabledState":true,"options":{}},{"id":2,"title":"Tech magazines","description":"Show tech news posts that are not related directly to programming, for example, reports about tech companies, startups, venture capital, and scientific discoveries.","group":"advanced","defaultEnabledState":true,"options":{}},{"id":3,"title":"Newsletters","description":"Show posts that were published on developer newsletters. Such posts usually contain curated lists and opinionated essays.","group":"advanced","defaultEnabledState":true,"options":{}},{"id":4,"title":"Product launches","description":"Show posts that aim to help you discover new product launches or major releases to existing developer tools.","group":"advanced","defaultEnabledState":true,"options":{}},{"id":5,"title":"Showcases","description":"Show posts that aim to showcase a project or other types of code snippets. These posts usually provide a demo-only of the showcased work without broader context around it.","group":"advanced","defaultEnabledState":true,"options":{}},{"id":6,"title":"Non-editorial content","description":"Show user-generated posts that were created on external blogging platforms. Such posts are usually not checked by professional editors for fact accuracy, spelling, grammar, and punctuation.","group":"advanced","defaultEnabledState":true,"options":{}},{"id":7,"title":"Videos","description":"Show video posts on my feed","group":"content_types","defaultEnabledState":true,"options":{"type":"video:youtube"}},{"id":8,"title":"Article","description":"Show article posts on my feed","group":"content_types","defaultEnabledState":true,"options":{"type":"article"}},{"id":9,"title":"Share","description":"Show share posts on my feed","group":"content_types","defaultEnabledState":true,"options":{"type":"share"}},{"id":10,"title":"Freeform","description":"Show freeform posts on my feed","group":"content_types","defaultEnabledState":true,"options":{"type":"freeform"}},{"id":11,"title":"Welcome","description":"Show welcome posts on my feed","group":"content_types","defaultEnabledState":true,"options":{"type":"welcome"}},{"id":12,"title":"Collection","description":"Show collection posts on my feed","group":"content_types","defaultEnabledState":true,"options":{"type":"collection"}}]
```

Comment:
I think smaller seed files, we can save them as expanded json?

```suggestion
[
  {
    "id": 1,
    "title": "Community recommendations",
    "description": "Show posts that are recommended by other community members.",
    "group": "advanced",
    "defaultEnabledState": true,
    "options": {}
  },
  {
    "id": 2,
    "title": "Tech magazines",
    "description": "Show tech news posts that are not related directly to programming, for example, reports about tech companies, startups, venture capital, and scientific discoveries.",
    "group": "advanced",
    "defaultEnabledState": true,
    "options": {}
  },
  {
    "id": 3,
    "title": "Newsletters",
    "description": "Show posts that were published on developer newsletters. Such posts usually contain curated lists and opinionated essays.",
    "group": "advanced",
    "defaultEnabledState": true,
    "options": {}
  },
  {
    "id": 4,
    "title": "Product launches",
    "description": "Show posts that aim to help you discover new product launches or major releases to existing developer tools.",
    "group": "advanced",
    "defaultEnabledState": true,
    "options": {}
  },
  {
    "id": 5,
    "title": "Showcases",
    "description": "Show posts that aim to showcase a project or other types of code snippets. These posts usually provide a demo-only of the showcased work without broader context around it.",
    "group": "advanced",
    "defaultEnabledState": true,
    "options": {}
  },
  {
    "id": 6,
    "title": "Non-editorial content",
    "description": "Show user-generated posts that were created on external blogging platforms. Such posts are usually not checked by professional editors for fact accuracy, spelling, grammar, and punctuation.",
    "group": "advanced",
    "defaultEnabledState": true,
    "options": {}
  },
  {
    "id": 7,
    "title": "Videos",
    "description": "Show video posts on my feed",
    "group": "content_types",
    "defaultEnabledState": true,
    "options": {
      "type": "video:youtube"
    }
  },
  {
    "id": 8,
    "title": "Article",
    "description": "Show article posts on my feed",
    "group": "content_types",
    "defaultEnabledState": true,
    "options": {
      "type": "article"
    }
  },
  {
    "id": 9,
    "title": "Share",
    "description": "Show share posts on my feed",
    "group": "content_types",
    "defaultEnabledState": true,
    "options": {
      "type": "share"
    }
  },
  {
    "id": 10,
    "title": "Freeform",
    "description": "Show freeform posts on my feed",
    "group": "content_types",
    "defaultEnabledState": true,
    "options": {
      "type": "freeform"
    }
  },
  {
    "id": 11,
    "title": "Welcome",
    "description": "Show welcome posts on my feed",
    "group": "content_types",
    "defaultEnabledState": true,
    "options": {
      "type": "welcome"
    }
  },
  {
    "id": 12,
    "title": "Collection",
    "description": "Show collection posts on my feed",
    "group": "content_types",
    "defaultEnabledState": true,
    "options": {
      "type": "collection"
    }
  }
]
```

Reply:
don't be so oblivious to git index size 😆

Reply:
I think it'll survive a few extra `\n` and ` `